index.php

```diff
@@ -77,7 +77,7 @@
     $templateData['custom_html'] = ob_get_clean();
 }
 
-if (LG_CHECK_LATENCY && filter_var(LookingGlass::detectIpAddress(), FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
```

Comment:
You can keep the `filter_var()` and just add the `FILTER_FLAG_IPV6` to it.

```suggestion
if (LG_CHECK_LATENCY && filter_var(LookingGlass::detectIpAddress(), FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 | FILTER_FLAG_IPV6)) {
```

Reply:
The filter_var can be removed, validation is done in `detectIpAddress()`. We added the filter to make sure it only works on v4 back then.